commit af80cf152159a4f680f0624e31c68c22a7933c25 (refs/changes/62/4562/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-07-24T15:08:45-07:00 (1 year, 2 months ago)
    
    TRITON-309 add new triton-origin-multiarch-18.1.0 image (reverts commit e99601f8dcdb0504e29b1eb561398449627f2e2f, manatee zone setup failure)

diff --git a/targets.json.in b/targets.json.in
index fb4b788..2168460 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -236,7 +236,7 @@ cat <<EOF
     "image_uuid": "b6ea7cb4-6b90-48c0-99e7-1d34c2895248",
     "pkgsrc": [],
     "repos": [
-        {"url": "git@github.com:joyent/triton-mockcloud.git", "dir": "mockcloud"}
+        {"url": "git@github.com:joyent/sdc-mockcloud.git", "dir": "mockcloud"}
     ],
     "public": false,
     "deps": [],
